android/app/build.gradle

```diff
@@ -117,7 +117,7 @@ android {
         versionCode 1
         versionName "1.0"
         ndk {
-            abiFilters "armeabi-v7a", "x86"
+            abiFilters "arm64-v8a", "x86"
```

Comment:
I remember when previously running we had needed all three instead of just two of them so something like

`"armeabi-v7a", "x86", "arm64-v8a"` just to keep it compatible, see this PR https://github.com/Skyscanner/backpack-react-native/pull/942

Reply:
Ok, added that back now 🙂